composer.json

```diff
@@ -20,9 +20,10 @@
         }
     ],
     "require": {
-        "php": ">=5.3",
+        "php": "5.3.*",
         "guzzle/guzzle": "dev-master#ecb935d2d0ecd8cddae4dcfb90515cac5e2cb023",
-        "psr/log": "1.0.0"
+        "psr/log": "1.0.0",
+        "justinrainbow/json-schema": "v1.6.0"
```

Comment:
This is a transitive dependency, but the version is not pinned, and by default composer will bring in a version that is too high to be compatible with 5.3.25. This is the highest version compatible with 5.3.25